### PR TITLE
add first task1

### PR DESCRIPTION
**Pull request by KrystsinaAlesiyuk**
Hi Jane! Could you please review it

### CODE REVIEW

File: .idea/java-script-study.iml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+<module type="WEB_MODULE" version="4">
+  <component name="NewModuleRootManager">
+    <content url="file://$MODULE_DIR$">
+      <excludeFolder url="file://$MODULE_DIR$/temp" />
+      <excludeFolder url="file://$MODULE_DIR$/.tmp" />
+      <excludeFolder url="file://$MODULE_DIR$/tmp" />
+    </content>
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+  </component>
```
Comment (YauheniyaYarmaliuk) [inline .idea/java-script-study.iml]: could you please delete it from git

File: .idea/libraries/Generated_files.xml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+        <item url="file://$PROJECT_DIR$/dist/wdio.conf.js" />
+      </sourceFilesUrls>
+    </properties>
+    <CLASSES>
+      <root url="file://$PROJECT_DIR$/dist/wdio.conf.js" />
+    </CLASSES>
+    <JAVADOC />
+    <SOURCES />
+  </library>
+</component>
```
Comment (YauheniyaYarmaliuk) [inline .idea/libraries/Generated_files.xml]: could you please delete it from git

File: .idea/modules.xml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectModuleManager">
+    <modules>
+      <module fileurl="file://$PROJECT_DIR$/.idea/java-script-study.iml" filepath="$PROJECT_DIR$/.idea/java-script-study.iml" />
+    </modules>
+  </component>
+</project>
```
Comment (YauheniyaYarmaliuk) [inline .idea/modules.xml]: could you please delete it from git

File: .idea/vcs.xml [hunk 4]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
+    <mapping directory="$PROJECT_DIR$" vcs="Git" />
+  </component>
+</project>
```
Comment (YauheniyaYarmaliuk) [inline .idea/vcs.xml]: could you please delete it from git

File: .idea/watcherTasks.xml [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+      </option>
+      <option name="outputFromStdout" value="false" />
+      <option name="program" value="$ProjectFileDir$/node_modules/.bin/babel" />
+      <option name="runOnExternalChanges" value="true" />
+      <option name="scopeName" value="Project Files" />
+      <option name="trackOnlyRoot" value="false" />
+      <option name="workingDir" value="$ContentRoot$" />
+      <envs />
+    </TaskOptions>
+  </component>
```
Comment (YauheniyaYarmaliuk) [inline .idea/watcherTasks.xml]: could you please delete it from git

File: test/features/main.feature [hunk 6]
```diff
@@ -0,0 +1,15 @@
+Feature: Sites Testing
+
+  Scenario: As a user, I can fill a new yahoo account
+    Given I am go to "https://login.yahoo.com/account/create" url
+    When I fill First name with "Kristina" first name
```
Comment (YauheniyaYarmaliuk) [inline test/features/main.feature]: refrase please

File: test/features/main.feature [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+    And I fill Last name with "Olesiyuk" last name
+    And I fill Email address with "krisolesiyuk_763" email
+    And I fill Password with "dfd5y65ngn" password
+    And I chose Country code with "BY" code
+    Then I expect Country code with "BY" code
```
Comment (YauheniyaYarmaliuk) [inline test/features/main.feature]: good!

File: test/features/main.feature [hunk 8]
```diff
@@ -0,0 +1,15 @@
...
+    When I fill Mobile phone Number with "291234567" number
```
Comment (YauheniyaYarmaliuk) [inline test/features/main.feature]: number twice

File: test/page-objects/yahoo.page.js [hunk 9]
```diff
@@ -0,0 +1,89 @@
+import Page from "./page";
+
+class NewYahooAccount extends Page {
+
+    get inputFirstName() {
+        return $('#usernamereg-firstName');
```
Comment (YauheniyaYarmaliuk) [inline test/page-objects/yahoo.page.js:6]: get firstName

File: test/page-objects/yahoo.page.js [hunk 10]
```diff
@@ -0,0 +1,89 @@
...
+    get inputEmailAddress() {
+        return $('#usernamereg-yid');
+    }
+
+    get inputPassword() {
+        return $('#usernamereg-password');
+    }
+
+    get choiceCode() {
+        return $('[name="shortCountryCode"]');
```
Comment (YauheniyaYarmaliuk) [inline test/page-objects/yahoo.page.js:22]: good!

File: test/page-objects/yahoo.page.js [hunk 11]
```diff
@@ -0,0 +1,89 @@
...
+        this.choiceCode.selectByAttribute('value', text);
+    }
+
+    setPhoneNumber(text) {
+        this.inputPhoneNumber.setValue(text);
+    }
+
+    setBirthMonth(text) {
+        this.choiceMonth.waitForExist();
+        this.choiceMonth.selectByVisibleText(text)
```
Comment (YauheniyaYarmaliuk) [inline test/page-objects/yahoo.page.js:76]: awesome!

File: test/page-objects/yahoo.page.js [hunk 12]
```diff
@@ -0,0 +1,89 @@
...
+    }
+
+    setBirthDay(text) {
+        this.inputBirthDay.setValue(text)
```
Comment (YauheniyaYarmaliuk) [inline test/page-objects/yahoo.page.js]: separate in method new metho
setText(el, text) => el.setValue(text);

**Review by YauheniyaYarmaliuk — COMMENTED** [review]
great job, with slite issues

File: .idea/java-script-study.iml (hunk 1 above)
Comment (KrystsinaAlesiyuk) [reply .idea/java-script-study.iml]: done

File: .idea/libraries/Generated_files.xml (hunk 2 above)
Comment (KrystsinaAlesiyuk) [reply .idea/libraries/Generated_files.xml]: done

File: .idea/modules.xml (hunk 3 above)
Comment (KrystsinaAlesiyuk) [reply .idea/modules.xml]: done

File: .idea/vcs.xml (hunk 4 above)
Comment (KrystsinaAlesiyuk) [reply .idea/vcs.xml]: done

File: .idea/watcherTasks.xml (hunk 5 above)
Comment (KrystsinaAlesiyuk) [reply .idea/watcherTasks.xml]: done

File: test/features/main.feature (hunk 6 above)
Comment (KrystsinaAlesiyuk) [reply test/features/main.feature]: done

File: test/features/main.feature (hunk 8 above)
Comment (KrystsinaAlesiyuk) [reply test/features/main.feature]: changed

File: test/page-objects/yahoo.page.js (hunk 9 above)
Comment (KrystsinaAlesiyuk) [reply test/page-objects/yahoo.page.js:6]: changed

File: test/page-objects/yahoo.page.js (hunk 12 above)
Comment (KrystsinaAlesiyuk) [reply test/page-objects/yahoo.page.js]: done

**Comment by hlebpapshau** [comment]
Hi @KrystsinaAlesiyuk! The code looks good, although it seems like the files in .idea/ are still in the branch.

**Pull request merged**